root/edit/search_macros.tt

```diff
@@ -357,6 +357,7 @@
                    [ 'not_subscribed', l('do not include editors in my subscriptions') ]
                    [ 'limited', l('include a beginner editor') ]
                    [ 'not_edit_author', l('include someone other than the edit author') ]
+                   [ 'nobody', l('do not exist (edit has no notes)') ]
```

Comment:
Maybe "include nobody" to match the language of the other options?

I was wondering if it'd be more useful to implement this as "Edit note count" or "Edit note author count" parameters instead (which you could specify 0 for), but this seems fine too.

Reply:
"edit notes include nobody" seems super confusing compared to "edit notes do not exist" though, which is why I did it this way originally - it at least feels like English :sweat_smile:

Reply:
It's "note authors include nobody" though, not "notes include nobody." :)

I think both sound kinda weird TBH, though we can always change the string later.

Reply:
Ah, true - both do sound weird (it's the crappy situation of trying to build "real sentence" edit searches, which also translate horribly - we should probably change the approach entirely).